WordPress/Info.plist

```diff
@@ -33,6 +33,16 @@
 				<string>${WPCOM_SCHEME}</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>com.optimizely</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>optly5635163020</string>
```

Comment:
Do you mind explaining your thoughts on why this is ok being in version control? I only ask because I decided against adding it in Android in the fear that something bad could happen, but I don't know what.